mozregression/download_manager.py

```diff
@@ -227,6 +229,10 @@ def __init__(self, destdir, session=requests, persist_limit=None):
         self.persist_limit = persist_limit or PersistLimit(0)
         self.persist_limit.register_dir_content(self.destdir)
 
+        # if folder does not exist, create it
+        if not os.path.isdir(destdir):
+            os.makedirs(destdir)
+
```

Comment:
looks all good to me here. Could you just adjust the comment to:
# if **persist**folder does not exist, create it
please ? that would allow to understand better why we are doing that, for when we will look at the code later.